auto3dseg/algorithm_templates/swinunetr/configs/hyper_parameters.yaml

```diff
@@ -1,66 +1,78 @@
-amp: true
 bundle_root: null
+ckpt_path: "$@bundle_root + '/model_fold' + str(@fold)"
+data_file_base_dir: null
+data_list_file_path: null
+fold: 0
+
+transforms:
+  resample_to_spacing: "$@training#resample_to_spacing"
 
 cache_rate: 1
 train_cache_rate: "$@cache_rate"
 validate_cache_rate: "$@cache_rate"
+show_cache_progress: false
 
-ckpt_path: $@bundle_root + '/model_fold' + str(@fold)
-use_pretrain: true
-pretrained_path: $@bundle_root + '/pretrained_model' + '/swin_unetr.base_5000ep_f48_lr2e-4_pretrained.pt'
-data_file_base_dir: null
-data_list_file_path: null
-determ: false
+training:
+  # hyper-parameters
+  amp: true
+  early_stop: true
+  early_stop_delta: 0
+  early_stop_patience: 20
+  determ: false
+  input_channels: null
+  learning_rate:  0.0001
+  log_output_file: "$@bundle_root + '/model_fold' + str(@fold) + '/training.log'"
+  num_images_per_batch: 1
+  num_epochs: 1 #1600
```

Comment:
This will break many tests

Reply:
Lots of tests use `num_epochs` to shorten the testing time. With such changes, all the tests need to use `training#epochs`. Same as all the other params

Reply:
Also it mean all the GitHub tutorials need to be updated to reflect such changes. This change of config file is actually a breaking change.

Reply:
@mingxin-zheng This template should be exactly the same with dints. I've updated the template to most recent dints template